coderd/coderd.go

```diff
@@ -478,12 +480,14 @@ type API struct {
 
 	derpServer *derp.Server
 
-	Handler             chi.Router
+	Auditor             atomic.Pointer[audit.Auditor]
+	RootHandler         chi.Router
+	APIHandler          chi.Router
```

Comment:
I understand why this needs to occur but it looks really sketchy and deserves a comment.